wof/examples/configs/timeseries/timeseries

```diff
@@ -0,0 +1,9 @@
+server {
+    listen 8080;
+    server_name 127.0.0.1;
+
+    location /odm2timeseries {
```

Comment:
If I change the application name to `wofpy` there will be a `wofpy.sock`. Assuming this is for 2 dao's not sure how to separate them within NGINX...